trafilatura/metaxpaths.py

```diff
@@ -11,7 +11,7 @@
     '//*[(self::a or self::address or self::link or self::p or self::span)][@rel="author" or @id="author" or @class="author" or @itemprop="author name" or rel="me"]|//author',
     '//*[(self::a or self::div or self::p or self::span)][contains(@class, "authors") or contains(@class, "authorName") or contains(@class, "author name") or contains(@class, "author") or contains(@class, "posted-by") or contains(@itemprop, "author")]',
     '//*[(self::a or self::div or self::p or self::span)][contains(@class, "byline") or contains(@id, "author") or contains(@class, "submitted-by")]',
-    '//*[contains(@class, "author") or contains(@class, "Author") or contains(@class, "screenname")]',
+    '//*[contains(@class, "author") or contains(@class, "Author") or contains(@class, "screenname") or contains(@data-component, "Byline") or contains(@itemprop, "author")]',
```

Comment:
`contains(@itemprop, "author")` is already present two lines above

Reply:
I put repeated because I found more elements like h3 with the itemprop, should I move it to bottom?

Reply:
I get it, that's fine, just put a comment in the code